packages/utils/src/error.ts

```diff
@@ -3,10 +3,13 @@ export class SentryError extends Error {
   /** Display name of this error instance. */
   public name: string;
 
-  public constructor(public message: string) {
+  public logLevel: string;
+
+  public constructor(public message: string, logLevel: string = 'warn') {
```

Comment:
```suggestion
  public logLevel: typeof CONSOLE_LEVELS[number];

  public constructor(public message: string, logLevel: typeof CONSOLE_LEVELS[number] = 'warn') {
```

Requires doing:

```ts
import { CONSOLE_LEVELS } from './logger';
```

I would even go as far as creating a type in `logger.ts` that looks like

```ts
export type ConsoleLevel = typeof CONSOLE_LEVELS[number];
```

just so we have this a bit more streamlined.

Reply:
Good idea. Done.